packages/react-components/react-avatar/SPEC-AvatarGroup.md

```diff
@@ -49,104 +66,105 @@ There are three layout variants in AvatarGroup:
 
 - Spread layout (Default): Avatars are spaced evenly.
 - Stack layout: Avatars are overlapped evenly.
-- Pie layout: For the pie layout there can be a minimum of two Avatars and a maximum of three. This layout does not overflow and provides a popover for more details.
-
-  - If the size of the avatar group is `36` or smaller then only the first letter of the initials will be displayed.
-  - `maxAvatars` will be ignored when using this layout.
-
-- The `spread` and `stack` layouts have a maximum of 5 avatars before overflowing by default, which can be overridden via the `maxAvatars` prop.
+- Pie layout: Avatars are "cut" in a pie design. When there are two Avatars inline, the Avatars will be cut in half and placed side by side. When there are three Avatars inline, the first Avatar will be cut in half and other two will be downscaled by 50%.
+  - The pie layout must have 3 or less Avatars inline and all Avatars should repeat in the `AvatarGroupPopover`. This is handled by `partitionAvatarGroupItems`.
+- If `partitionAvatarGroupItems` is used, the `spread` and `stack` layouts should have a maximum of 5 avatars before overflowing by default. This can be overridden via the `maxAvatars` option in `partitionAvatarGroupItems`.
 
 ## API
 
-See [AvatarGroup.types.ts](./src/components/AvatarGroup/AvatarGroup.types.ts) for more details.
+See [AvatarGroup.types.ts](./src/components/AvatarGroup/AvatarGroup.types.ts), [AvatarGroupPopover.types.ts](./src/components/AvatarGroupPopover/AvatarGroupPopover.types.ts) and [AvatarGroupItem.types.ts](./src/components/AvatarGroupItem/AvatarGroupItem.types.ts) for more details.
 
 - `size`: Group size will override the children's current size. This is to ensure that the `AvatarGroup`'s spacing is correct because it changes depending on the group size.
-- `popoverSurface`: All Avatars in `popoverSurface` will have a size of 24 and will be encased in a div to apply stylings.
-- Avatar `color`: AvatarGroup's colors will follow the order below, but they can be overriden by providing a color specific color on a given avatar.
+- `AvatarGroupPopover`: All Avatars in `AvatarGroupPopover` will have a size of 24 and have a wrapper to apply stylings.
+- AvatarGroupItem `color`: Can be overridden by providing a specific color on a given avatar.
 
 #### Color override example:
 
-In this example, the first Avatar will have a `darkRed` color, while all the other avatars will follow the default color order.
+In this example, the first AvatarGroupItem will have a `darkRed` color, while all the other Avatars have their color assigned by Avatar.
 
 ```jsx
 <AvatarGroup>
-  <Avatar color="darkRed" name="Katri Athokas" />
-  <Avatar name="Elvia Atkins" />
-  <Avatar name="Cameron Evans" />
-  <Avatar name="Wanda Howard" />
-  <Avatar name="Mona Kane" />
+  <AvatarGroupItem color="darkRed" name="Katri Athokas" />
```

Comment:
Are we still supporting this? I remember a conversation where we discussed dropping it because the color scale is meant to be applied in a certain order.

Reply:
Since we are getting rid of the color order, this is still supported. We decided to let Avatar apply the colors.

I think @smhigley mentioned that the color contrast between two Avatars next to each other is not an a11y concern. That's why we dropped the color order.